release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1464,6 +1476,18 @@ In this update, `systemd` service only sets the default RPS mask for virtual int
+
+* Previously, {product-title} 4.8 added an API for customizing platform routes. This API includes status and spec fields in the cluster ingress configuration for reporting the current host names of customizable routes as well as the user's desired host names for these routes, respectively. The API also defined constraints for these values. These constraints were restrictive and excluded some valid potential host name. Consequently, the restrictive validation for the API prevented users from specifying custom host names that should have been permitted, and prevented users from being able to install clusters with domains that should have been permitted. With this update, the constraints on host names were relaxed to allow all host names that are valid for routes and the {product-title} allows users to use cluster domains with TLDs that contain decimal digits. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039256[*BZ#2039256*])
+
+* * Previously, the Ingress Operator did not check whether Ingress Controllers with spec.domains matched the spec.baseDomain. This caused the Operator to create DNS records and set `DNSManaged` conditions to false. With this fix, the Ingress Operator now checks whether the spec.domain matches with the cluster's spec.baseDomain. As a result, the Ingress Operator does not create DNS records and sets `DNSManaged` conditions to false for custom Ingress Controllers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2041616#[*BZ#2041616*])
+
+* Previously, in OpenShift Container Platform 4.10, the HAProxy must-gather function could take up to an hour to run. This can happen when routers in the terminating state delay the `oc cp` comand. The delay lasts until the pod is terminated. With the new release, a 10 minute limit on the `oc op` command prevents longer delays.   (link:https://bugzilla.redhat.com/show_bug.cgi?id=2104701[*BZ#2104701*])
+
+* Previously, the Ingress Operator did not clear the route status when Ingress Controllers were deleted, showing that the route was still in the operator after its deletion. This fix clears the route status when an Ingress Controller is deleted, resulting in the route being cleared in the operator after its deletion. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1944851#[*BZ#1944851*])
+
+* Previously, the OC explain route status showed 'Currently only Ready' rather than 'Admitted' due to incorrect wording in the Application Programming Interface (API). This fix updates the route status from 'Currently only Ready' to 'Admitted', resulting in the correct wording in the API. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2041133#[*BZ#2041133*])
```

Comment:
s/the OC explain route status/the output for the `oc explain router.status.ingress.conditions` command (that is the only command that appears to have been changed)
This fix corrects the wording in the API. As a result, the command output is correct.  (Not sure you need to incluse _from 'Currently only Ready' to 'Admitted'_. Seems obvious to me.